x11-libs/fltk/fltk-1.3~git.recipe

```diff
@@ -127,6 +122,7 @@ INSTALL()
 	packageEntries devel \
 		$developDir \
 		$binDir \
+		$docDir \
```

Comment:
I have a feeling that these should better be in `developDocDir`

Reply:
Moved them to $developDocDir, guess this makes the patch obsolete?

Reply:
Patch still needed, but it's ok now :)